setup.py

```diff
@@ -98,7 +98,7 @@ def txt_read(*paths):
         "matplotlib>=1.5",
         "cartopy[plotting]",
         "h5py",
-        "wxPython==3.0.2",
+        "wxPython>=3.0.2",
```

Comment:
I basically tried to refresh my initial packaging work on hdf-compass, for which I was using a snapshot of the git repository from a couple of months ago. I rebased the packaging work onto `v0.6.0b3` and ran the packaging build again. The setuptools dependency is the main blocker so far. Last time I checked, it ran fine with the wxpython available in Debian.